### PR TITLE
Ouster ros separation

### PR DESCRIPTION
**Pull request by Samahu**
## Summary of Changes
* Move `ouster-ros` into a separate repo
* Update `CMakeLists.txt` accordingly
* Update documentation accordingly

## Validation
N/A

## Linked PR(s)
* https://github.com/ouster-lidar/ouster-ros/pull/1

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -3,6 +3,14 @@ Changelog
 =========
 
 
+[202210xx]
+==========
```
Comment (kairenw) [inline CHANGELOG.rst:7]: Generally we use unreleased, but it's okay if you wanna leave it this way. My guess is 202211xx for the actual next release :wink:

**Review by kairenw — APPROVED** [review]

File: CHANGELOG.rst (hunk 1 above)
Comment (Samahu) [reply CHANGELOG.rst:7]: Oh, I thought these were change log entry dates, not release tags. I can fix that. I just remove the date entry, right?

Comment (kairenw) [reply CHANGELOG.rst:7]: Yah and replace with "Unreleased"

Comment (Samahu) [reply CHANGELOG.rst]: addressed

**Pull request merged**